reconcilers/solver/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM quay.io/thoth-station/thoth-ops:v0.15.0
```

```suggestion
FROM quay.io/thoth-station/thoth-ops:latest
```
As every time we would update thoth-ops, we will redirect latest to the updated version.